lib/adapters/msa.js

```diff
@@ -26,7 +26,7 @@ module.exports = adapter = View.extend({
         delete e.collection;
       });
 
-      this.msa = new msa(props);
+      this.msa = new msa.msa(props);
```

Comment:
that's actually a bug in the msa viewer - https://github.com/wilzbach/msa/issues/144
(that being said - thanks for fixing)